### DISCUSSION TITLE
The First Town Hall

### DISCUSSION THREAD

**Discussion by vovacodes** [Developer-DAO/community#9] [General]
We are a couple of weeks in, quite a bit of folks joined recently, so we think it's time to get together as a the community and chat a bit about what's this place.

Here's a suggested agenda to get things going. Interested hear what everyone would like to cover so comment below and we can amend/update the agenda:

- What is the phenomenon of DAO.
- The history of Developer DAO.
- State of current affairs: 
  - Current projects/collaborations
  -  Discussions happening in the community.
- What are the most important questions we'd like to work out as a community
- Future planning
  - Near term
  - Mid term
  - Long term  
- Call to action: encourage starting events, collaboration, etc. Let folks promote their projects and invite contributors.

I was thinking we could each take responsibility for covering one of these points, so we need volunteers.

Organizers: @johnny-liaw @kempsterrrr @vovacodes

**Comment by Dhaiwat10** [comment]
I'd like to volunteer & handle the call to action bit re. collabs & projects!

**Reply by tawseefnabi** [reply]
Im intrested and would like to volunteer here

**Comment by kempsterrrr** [comment]
Yes ! Excited to get this moving :)

For me the most important things to begin addressing are:

1. Our **WHY** or our Mission, values and goals of our community (at least early version that is documented)
2. An agreed process for proposals and voting (simple governance)  
3. A way to recognise and reward contributions

I think together these points will really help the community move forward as a group :)

Love to know everyone else's thoughts !!!

**Comment by jsjoeio** [comment]
Chiming in to say as someone new to the community, I would love to know how I can participate in the community here and there (especially if I struggle to participate in the Discord because it feels overwhelming 😅).

i.e. 
- monthly events
- weekend collaborations
- etc.

**Reply by kempsterrrr** [reply]
Great shout Joe. This came up a bit in the impromptu meet on Saturday - check the notes [here](https://github.com/Developer-DAO/community/blob/main/town-hall/2021-09-18.md).

Two early things to start addressing this were discussed and agreed:

1. Regular weekly DAO meeting to update on happenings/projects and discuss broader topics in the DAO.
2. Every project organise their own regular weekly meeting so people can join to either just listen or start to participate
3. Figure out a way to publicly share a calendar of DAO activities so people can know when/where they can find out more about whatever it is they're interested in in.

Good to know if this what you meant and any more ideas you may have about other ways to make this better :)

**Reply by jsjoeio** [reply]
Thanks for taking notes! Looked over them briefly.

Yes!

> Figure out a way to publicly share a calendar of DAO activities so people can know when/where they can find out more about whatever it is they're interested in in.

This sounds like what I was hoping for. That way, I can see what's happening and join an event or two per month as time permits. Thanks for following up! 🙌

**Reply by kempsterrrr** [reply]
Welcome, good to know the group were thinking along the right lines! Hopefully have it sorted by the end of the week 👍🏻

**Comment by teeolendo** [comment]
Would like to add an agenda item on training. I'm very interested in curating/creating training content to onboard as many Web2 engs to Web3. There's a lot of content out there but there isn't enough in terms of a roadmap that could help a developer start from a given point to being "hire-able". I think the DAO, given our numbers, can help build a talent pipeline to/for Web3 and I'd like to tease out the beginning steps towards that in the town hall.

**Reply by kempsterrrr** [reply]
Love this. Came up in the meet on Saturday regarding collecting resources and figuring out a way to help everyone learn. @with-heart has created a new repo call `resources` to get this going but think we need volunteers to take on getting it organised. Is that something you'd be keen to tackle @teeolendo ?
